unity/Hello_World/Assets/Scripts/TaskManager.cs

```diff
@@ -59,15 +59,17 @@ protected void Start()
     {
         Logger log = logger();
         TaskLogger taskLog = taskLogger();
-
-
-        TaskUpdateMsg taskUpdateMessage = new TaskUpdateMsg(new HeaderMsg(),
-                                               "Waiting for task",
-                                               new string[0],
-                                               "N/A",
-                                               "N/A",
-                                               "N/A",
-                                               "N/A");
+        TaskUpdateMsg taskUpdateMessage = new TaskUpdateMsg(new HeaderMsg(), // ROS std header
```

Comment:
I was confused for a second on seeing this in the C#. A comment here along the lines of "this is a placeholder to provide initial messaging in the UI" would be helpful.